assessment.md

```diff
@@ -93,9 +93,50 @@ Project Management is Covered by Team 2
 ### Interactions
 
 * Interaction with the customer
+  * :disappointed: Disregards the customer's feelings and
+  refuses to take their needs into account
+  * :confused: Regards some of the customer's feelings
+  but fails to take their ideas and needs into account
+  * :neutral_face: Regards most of the customer's feelings
+  and takes some of their ideas and needs into account
+  * :smiley: Regards all of the customer's feelings and
+  incorporates most of their ideas and needs while also communicating challenges
+  * :sunglasses: Regards all of the customer's feelings, improves ideas,
+  incorporates all of their needs, clearly communicates challenges and
+  provides updates on progress
```

Comment:
Hello @schultzh and @MaddyKapfhammer! This content is very well written and easy to understand. Thanks for your hard work on this project, it is greatly appreciated!

I noticed that none of your writing uses a comma before the last "and". Is that a writing decision that you and all of the other team members plan to enforce? While I recognize that this is a _very_ small point, it might be worth deciding whether or not you are going to use the "Oxford comma" or if you are going to omit it in all of your technical writing.

Reply:
I added the "Oxford comma" to all of our technical writing to solve this issue. Thank you for raising this point!